### ISSUE TITLE
[Relay Theory 0.1 Formalization] Stochastic Recoverability Preorder — can valid garbling define an atemporal information arrow?

### ISSUE THREAD

**Issue by rinsakamo** [rinsakamo/relay-lm#2548]
# Status / scope

Child formalization lane of #2532 F0/F1/F2 on authority branch `relay-theory`.

Fresh authority at creation:

```text
relay-theory HEAD = dc9af1bd9281528f3c6d82884ee02becf2d6c3c1
tree              = 2302df073afa00d8d5a3fd653b19c46c05810ead
```

Predecessor #2543/#2544 established only that a family of local scalar chains can encode a pre-supplied non-total causal order. This lane removes the word `time` entirely and asks whether an operational directed relation can instead be induced from physically admissible stochastic recoverability itself.

No RelayLM `v1`/`v2` runtime or product mutation is authorized.

# Classical prior-art boundary

The basic comparison is classical Blackwell / garbling order territory: one statistical experiment/channel is no more informative than another when it can be obtained by stochastic post-processing of the latter. Markov-category generalizations also exist.

This lane claims no novelty for that preorder. Relay-specific purpose is to connect the already-earned exact-vs-stochastic recovery boundary to #2532's falsification of `recoverability loss = temporal orientation`.

Relevant prior art:
- Blackwell–Sherman–Stein / comparison of statistical experiments.
- Rauh et al., *Coarse-graining and the Blackwell order*, arXiv:1701.07602.
- Fritz et al., *Representable Markov Categories and Comparison of Statistical Experiments in Categorical Probability*, arXiv:2010.07416.
- Buscemi & Datta, *Equivalence between divisibility and monotonic decrease of information in classical and quantum stochastic processes*, arXiv:1408.7062.

# Grand Null

> Once temporal language is removed, stochastic-valid recoverability does not yield a coherent directed information order suitable even as a candidate arrow: reflexivity/transitivity fail, cycles destroy directionality, or exact algebraic reversibility collapses every apparent loss.

The Null loses only to mechanically earned finite exact structure. Even then, the result is an **information order**, not yet time.

# G0 — common-source stochastic garbling

For observation kernels with the same latent source,

```text
strong : FinKernel n qa
weak   : FinKernel n qb
```

define:

```text
StochasticDegradesTo strong weak :=
  exists post : FinKernel qa qb,
    Valid post /\
    compose post strong = weak
```

Orientation discipline:

```text
strong -> weak
```

means `weak` is reproducible from `strong` by physically admissible stochastic post-processing. No `time`, `earlier`, `later`, entropy gradient, or supplied causal order may occur in this definition.

# G1 — preorder test

Mechanically prove or refute:

```text
reflexive
transitive
```

using valid identity and valid composition.

If either fails, stop and record the counterexample rather than patching the definition merely to obtain an order.

# G2 — raw antisymmetry pressure

Test literal kernel equality antisymmetry. Expected negative control:

```text
identity observation
<->
valid deterministic flip observation
```

They should be mutually garblable while remaining unequal kernels.

If earned, record:

```text
RAW_KERNEL_ANTISYMMETRY_REJECTED
```

and define mutual stochastic equivalence separately rather than pretending the raw preorder is a partial order.

# G3 — strict stochastic information loss

Define the strict part:

```text
StrictStochasticLoss A B :=
  StochasticDegradesTo A B /\
  not StochasticDegradesTo B A
```

Pressure at minimum:

```text
irreflexive
transitive
asymmetric
```

Do not claim totality; incomparability is permitted and expected.

# G4 — exact algebraic vs stochastic-valid reversal

Reuse the existing noisy binary observation from `EpicityReversibility`:

```text
finNoisyEpic2
```

Already earned:

```text
Valid finNoisyEpic2
HasExactRecovery finNoisyEpic2
not HasValidStochasticRecovery finNoisyEpic2
```

Test mechanically:

```text
identity 2 -> finNoisyEpic2
```

by valid garbling, but not the reverse. At the same time retain its signed exact algebraic inverse.

This is the critical F1 discriminator:

```text
exact algebraic invertibility
!=
stochastic information equivalence
```

# G5 — reversible stochastic control

Use deterministic binary flip as a positive reversible control:

```text
identity 2 <-> dirac flip
```

with valid stochastic maps both ways. Strict loss should be false in both directions.

# G6 — cycle interpretation

Mutual garbling creates cycles in the raw preorder, but these are equivalence cycles rather than strict-loss cycles. Distinguish:

```text
preorder cycle
strict-loss cycle
```

Do not infer temporal recurrence or physical reversibility from preorder equivalence alone.

# G7 — temporal interpretation boundary

Even if the strict relation is a well-behaved atemporal information order, it does **not** yet establish:

```text
information order = time order
physical dynamics is monotone in that order
an arrow orientation is selected without boundary conditions
Past Hypothesis is unnecessary
experienced 1D time has been derived
```

A later lane must place an actual process/path through observation states and test whether physically allowed evolution is monotone in the strict order, cyclic, incomparable, or observer-dependent.

# Acceptance

1. no temporal variable or temporal wording inside the operational relation definition;
2. stochastic-valid garbling relation on common-source finite kernels;
3. reflexivity and transitivity mechanically proved or explicitly refuted;
4. literal antisymmetry pressure with an explicit reversible-but-distinct control;
5. mutual stochastic equivalence separated from strict loss;
6. strict-loss irreflexivity/transitivity/asymmetry mechanically tested;
7. noisy exact-inverse / no-valid-recovery control connected to strict stochastic loss;
8. deterministic flip reversible control;
9. strict claim boundary: information order is not yet time;
10. no `sorry`, Lean `admit`, `native_decide`, or substantive axiom;
11. no unnecessary Matrix/rank/determinant/category dependency;
12. exact-head and merge-head Lean build/kernel-check/axiom-audit + regular Relay Theory CI GREEN.

# Candidate scoped verdicts

Only if mechanically earned:

```text
STOCHASTIC_GARBLING_PREORDER_EARNED
RAW_KERNEL_ANTISYMMETRY_REJECTED
STRICT_STOCHASTIC_INFORMATION_LOSS_IS_IRREFLEXIVE_TRANSITIVE_ASYMMETRIC
EXACT_ALGEBRAIC_INVERTIBILITY_DOES_NOT_ERASE_STRICT_STOCHASTIC_LOSS
REVERSIBLE_STOCHASTIC_EQUIVALENCE_HAS_NO_STRICT_ARROW
ATEMPORAL_INFORMATION_ORDER_IS_NOT_YET_TIME_ORDER
PROCESS_MONOTONICITY_REMAINS_REQUIRED_FOR_TEMPORAL_INTERPRETATION
```

Refs #2532 #2543 #2544 #2500

**Comment by rinsakamo** [comment]
Completed via PR #2550.

Fresh merged authority:

```text
relay-theory HEAD = c7d108057143472667cb08d4ac115995b13cef46
tree              = 8beac4e302ffa3fede09d494746a527ea9b02b5b
```

Mechanically earned on the exact finite-kernel interface:

```text
STOCHASTIC_GARBLING_PREORDER_EARNED
RAW_KERNEL_ANTISYMMETRY_REJECTED
STRICT_STOCHASTIC_INFORMATION_LOSS_IS_IRREFLEXIVE_TRANSITIVE_ASYMMETRIC
EXACT_ALGEBRAIC_INVERTIBILITY_DOES_NOT_ERASE_STRICT_STOCHASTIC_LOSS
REVERSIBLE_STOCHASTIC_EQUIVALENCE_HAS_NO_STRICT_ARROW
ATEMPORAL_INFORMATION_ORDER_IS_NOT_YET_TIME_ORDER
PROCESS_MONOTONICITY_REMAINS_REQUIRED_FOR_TEMPORAL_INTERPRETATION
```

Core definition contains no time coordinate or causal relation:

```text
A -> B
iff
exists valid stochastic post-processing K,
  B = K compose A
```

The raw relation is reflexive/transitive but not antisymmetric on literal kernels: binary identity and deterministic flip are mutually garblable while unequal. Mutual simulation is therefore separated as stochastic equivalence.

Its strict part `A -> B and not B -> A` is mechanically irreflexive, asymmetric, and transitive.

Critical F1 control:

```text
identity 2 -> finNoisyEpic2
not (finNoisyEpic2 -> identity 2)
```

so the noisy binary channel is a strict stochastic information loss from identity, even though its previously earned signed exact algebraic inverse still exists. Thus exact algebraic invertibility does not erase the physically admissible stochastic-loss distinction.

Verification:
- PR head `24833c862b745d2dca90f1719aa42b2324a0c3f7`: Lean build/kernel-check/axiom-audit GREEN; regular CI 4/4 GREEN.
- merge head `c7d108057143472667cb08d4ac115995b13cef46`: Lean build/kernel-check/axiom-audit GREEN; regular CI 4/4 GREEN.
- complete patch audit: no `sorry`, Lean `admit`, `native_decide`, substantive axiom, or Matrix/rank/determinant/category-library dependency.

This remains an atemporal information order, not a derivation of time. Actual process monotonicity, cyclic/regaining dynamics, observer-relative admissibility, and arrow orientation remain open.

Acceptance satisfied; closing completed.
